start.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+my_dir="$(dirname "$0")"
```

Comment:
The installation directory is already defined by environment variable `CONF_INSTALL` and would be a reliable variable to use.